make/devices.cmake

```diff
@@ -771,6 +771,7 @@ function(ADD_FPGA_TARGET)
   endforeach()
 
   get_target_property_required(VPR env VPR)
+  get_target_property_required(GENHLC env GENHLC)
```

Comment:
Need to get_target_property GENHLC_TARGET and add ${GENHLC_TARGET} to the add_custom_command target list.

Reply:
I think I did this correctly. Any recommended way to check it?

Reply:
Not really.  If we ever break the parts of verilog -> bit flow up, it would be testable.  But right now GENHLC_CMD depends on an output that depends on VPR_TARGET, and VPR_TARGET == GENHLC_TARGET.  However this is now correct.